qml/Stage/Stage.qml

```diff
@@ -784,10 +789,15 @@ FocusScope {
                 Binding {
                     target: model.window; property: "requestedPosition"
                     // miral doesn't know about our window decorations. So we have to deduct them
-                    value: Qt.point(appDelegate.requestedX + appDelegate.clientAreaItem.x,
-                                    appDelegate.requestedY + appDelegate.clientAreaItem.y)
+                    value: Qt.point(appDelegate.requestedX + appDelegate.clientAreaItem.x + screenOffsetX,
+                                    appDelegate.requestedY + appDelegate.clientAreaItem.y + screenOffsetY)
```

Comment:
You may want to comment here that "the position is relative to Mir's global coordinate i.e. screen offset is required".

Reply:
Will add some comments